### PR TITLE
Update makecpt baseline images for GMT 6.2.0rc1

### PR DESCRIPTION
**Pull request by meghanrjones**
**Description of proposed changes**

Address #1217

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Review by seisman — APPROVED** [review]

**Review by weiji14 — APPROVED** [review]

**Review by weiji14 — CHANGES_REQUESTED** [review]
Actually, hold on, the `makecpt` tests are still failing https://github.com/GenericMappingTools/pygmt/pull/1231/checks?check_run_id=2414367157#step:11:687:

```
___________________________ test_makecpt_categorical ___________________________
Error: Image dimensions did not match.
  Expected shape: (122, 1432)
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_makecpt.test_makecpt_categorical/baseline.png
  Actual shape: (120, 1432)
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_makecpt.test_makecpt_categorical/result.png
_____________________________ test_makecpt_cyclic ______________________________
Error: Image dimensions did not match.
  Expected shape: (122, 1529)
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_makecpt.test_makecpt_cyclic/baseline.png
  Actual shape: (120, 1528)
    /Users/runner/work/pygmt/pygmt/tmp-test-dir-with-unique-name/results/pygmt.tests.test_makecpt.test_makecpt_cyclic/result.png
```

**Comment by meghanrjones** [comment]
Locally it passes for me with `pytest pygmt/tests/test_makecpt.py` but fails with `make test`. I will need to look into this more but do not have time right now. I will tomorrow but if anyone else wants to take this over to finish quicker that is fine.

File: pygmt/tests/test_makecpt.py [hunk 1]
```diff
@@ -150,6 +150,7 @@ def test_makecpt_continuous(grid):
     return fig
 
 
+@pytest.mark.xfail(reason="Flaky test only passes with pytest on single module")
```
Comment (weiji14) [inline pygmt/tests/test_makecpt.py:153]:
```suggestion
@pytest.mark.xfail(
    reason="Flaky test only passes with pytest on single module"
    "See https://github.com/GenericMappingTools/pygmt/issues/1242"
)
```

File: pygmt/tests/test_makecpt.py [hunk 2]
```diff
@@ -161,6 +162,7 @@ def test_makecpt_categorical(region):
...
```
Comment (weiji14) [inline pygmt/tests/test_makecpt.py:165]:
```suggestion
@pytest.mark.xfail(
    reason="Flaky test only passes with pytest on single module"
    "See https://github.com/GenericMappingTools/pygmt/issues/1242"
)
```

**Review by weiji14 — COMMENTED** [review]
Ad a URL to the PyGMT issue to keep track.

**Review by weiji14 — APPROVED** [review]

**Pull request merged**